### DISCUSSION TITLE
Using 60V battery, + rpm question in src file.

### DISCUSSION THREAD

**Discussion by WvBijsterveld** [danielnilsson9/bbs-fw#183] [Q&A]
Hello!

I'm going to build myself a new battery for the first time, i was planning on going 48V but realised i could go 60v with my cell amount and maybe go for a small 12v battery. (for powering external lights, not relevant)

 Since i remembered you can hook up a 60v battery (!using the software! correct?) do you need to scale the amps down with it? i know the motor can run the 1800/2000W fine but maybe same as BBS02 it "realizes" that it's above some set parameter?
But i have no idea on the controller.
 
 on a sort of seperate note, the max rpm signal thing(don't know the exact thing but it was in source files (not plain rpm, it was 4780 out of my head) does adjusting that increase the motor rpm or is that just the defined max rpm for calculations and functions?
 Also does this need to be changed when on 60v? (i think rpm increase slightly with more volts)
 
Oh yea is anyone running 60v or has run/tested 60v? let me know your experience! 

Looking forward to any answers and comments!

**Comment by silenec** [comment]
60V battery is like 67V fully charged. Caps in the controller are rated at 63V I think. 52V battery should be fine.

**Reply by WvBijsterveld** [reply]
and if i would temp set the bms to hvc 63v? or even 65v because voltage sag during load?

the display is rated for 60v, what would even happen when going above the 63v threshold? would it be "a gamble" or just instantly die/shut off?

**Comment by zeynelozturk** [comment]
A 15S battery is 63V fully charged (which could be my next manually build battery). The caps should be ok I believe, as it won't ve even 63V on the bike, probably rather 62.9 or something... I'm curious if anyone has any experience with such battery & BBS-FW combination.

**Reply by WvBijsterveld** [reply]
im all ears for the answers!

**Reply by skeeterman007** [reply]
15s battery at 28amps would probably work fine just watch your temps and maybe upgrade the nylon gear to a peek gear, also you'd have to use the egg rider display or 750c display for anything over 60v, most displays wont work or even break as soon as you run anything over 60v. with less amps you'll still get a pretty similar top speed since volts are what increase the RPM of the motor. but if you want to run a 60v battery you'd have to upgrade the controller or do a rather difficult mod on the stock controller.

**Reply by WvBijsterveld** [reply]
Thanks for the info and reply, what information is your comment based on? is there a limiter that you cant run above 1600W on stock controller or why do you say 28 A? this gives less W then 48v on 33A but it's slightly less so will probably make up for it in RPM but still why not 30 or 33?
the display i am using (B02N-bluetooth) supports 60V.

**Reply by skeeterman007** [reply]
the main reason id suggest running less amps is so that theres less current running through the controller (and less heat as higher amps tend to create more heat compared to volts from what i know) as they arent exactly designed to handle upwards of 2000w but to be fair ive never tested the stock controller with more than 60v so it may not be an issue to run 33 amps 63v but if you wanted to play it safe and have your controller last a bit longer running less amps e.g. 28 would only put a max of 1700w through vs 2000 and youd still get very similar performance (within 4kmh of the top speed 63v 33 amp would get you). you could also consider shunt modding your controller though and then you could run up to 50 amps and the controller could probably handle it better. but i would try experimenting with different amperage's and seeing how much more heat it gives off and how much of a difference in torque and top speed it makes. also if you can find a good 15s battery please share it with me as ive been searching for one for a while and being able to run 63v would make a huge difference youd probably be able to get up to 70 on flat. (and from what i know theres no hardware limit on the wattage that can be output but your display may not be able to read it to a certain extent depending on the display but regardless of that the motor should be able to run at pretty much any wattage if the controller and nylon gear can handle it)

**Reply by WvBijsterveld** [reply]
okok perfect cool i think i will build it 15s then, gonna try it out 30a, work to 33a then see what is what.
i've heard of the shunt mod but i wanna build this battery to use with my stock controller and maybe upgrade to the high voltage kit (ASI 855 controller, could be wrong number but you get it) where my you can hook up 100v battery's. so want to create a battery with as high volts as i can use with both controllers, atleast i want to use my newly build battery and not have to wait.

Either way no i'm sorry i don't know any 15s battery's. im building my own i got 104 21700 cells and was firstly going for a 13s8p build but the heat and rpm is gonna be the drawback where i want to build a higher voltage one.

my current display already shows W's pretty useless so doesn't matter if i go higher lol.

Will build battery around next week so will report back then!

Still debating with myself if i wanna go 16s battery and then limit the charging to 63v for the time being. hmm...

**Reply by zeynelozturk** [reply]
Please let us know if the display still works at 63V :)  It may be even possible to drop the voltage to the display (via in series diodes), but I don't know if that'd be necessary.

**Reply by WvBijsterveld** [reply]
definitely will, the manual for my display display says it supports 60v so i will contact the store when it stops functioning (lol).

**Reply by Jaaa67** [reply]
Hi, I tested the ekd01 display with a lab bench power supply up to 62V.
Everything worked fine with a 15s battery on the bbsdh.
But to be safe I would prefer to use a 14s :)

I also tested a 36v 10s battery just out of curiosity and it works pretty well, but it only makes about 1200w max with the bbshd.

**Comment by danielnilsson9** [comment]
Max cadence RPM set in fwconfig.h is only used for computing "keep current" function and it's not that important that its 100% correct. It's not used for any motor control.
